buildSrc/build.gradle

```diff
@@ -3,24 +3,26 @@ import groovy.xml.XmlUtil
 build {
 	doLast {
 		def ideaFolder = new File('.idea');
-		if(ideaFolder.exists() && ideaFolder.isDirectory()) {
+		if (ideaFolder.exists() && ideaFolder.isDirectory()) {
 			def xmlFile = new File(".idea/vcs.xml")
-			def xml = new XmlSlurper().parse(xmlFile)
-			xml.component.find {it.@name == 'IssueNavigationConfiguration'}.replaceNode {}
-			xml.appendNode {
-				component(name: "IssueNavigationConfiguration") {
-					option(name: 'links') {
-						list {
-							IssueNavigationLink {
-								option(name: "issueRegexp", value: '#(\\d+)')
-								option(name: "linkRegexp", value: 'https://github.com/junit-pioneer/junit-pioneer/issues/$1')
+			if (xmlFile.exists() && xmlFile.isFile()) {
```

Comment:
You added this check, right? Can you let us know why it was necessary? If we need to change this file, we should so it in a separate change.

Reply:
I did indeed. It was necessary because when I imported this project into my IntelliJ IDEA, the file `.idea/vcs.xml` was absent for some bizarre reason. So whenever I tried to run any of the Gradle tasks, it threw an exception upon trying to parse the file.

That's a good shout, putting this in a separate change. Given I'm busy IRL, is this something that someone from the core team would be able to handle? If not, then that's fine, as I've just put it in my TODO list.